apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -442,6 +443,14 @@ export class Utilities {
     return Utilities._executeLifecycleCommandInternal(command, child_process.spawn, options);
   }
 
+  public static isNonDebugTabCompletionRequest(): boolean {
+    return process.argv.length > 2 && process.argv[2] === CommandLineConstants.TabCompletionActionName;
```

Comment:
Can we put this whole function in `ts-command-line`? Seems like anyone who would be using `CommandLineConstants.TabCompletionActionName` would end up basically rewriting this whole expression.

Reply:
Good point.